category-committed.php

```diff
@@ -0,0 +1,313 @@
+      .committed-archive__logo {
+        height: 150px;
+      }
+      .committed__serif-large {
+        font-size: 19px;
+      }
+      .committed__serif-medium {
+        font-size: 16px;
+      }
+      .committed-credits__background {
+      background-image: url(<?php echo $base_image_path . 'committed-credits-background-m.png'; ?>);
+    }
+
+    }
+
+    @media screen and (max-width: 759px) {
+      .committed-archive__logo {
+        height: 70px;
+      }
+      .committed-archive__subtitle {
+        height: 50px;
+       }
+      .committed__serif-large {
+        font-size: 18px;
+      }
+      .committed-credits__background {
+        background-image: url(<?php echo $base_image_path . 'committed-credits-background-s.png'; ?>);
+      }
+      .committed-credits__logo {
+      height: 45px;
+    }
+    .committed-credits__reporters img{
+      height: 34vw;
+    }
+    }
+     @media screen and (max-width: 480px) {
+        .committed-listen-now-title{
+        height: 1.3rem;
+      }
+     }
+  </style>
+<div class="committed-archive__container committed__backgrounded">
+  <section class="container">
+    <div class="flex-grid-row mb-4">
+      <div class="flex-grid-item flex-item-xxl-12 flex-item-m-10 mb-4 mt-4">
+        <a href="<?php echo home_url( 'category/audio/' ); ?>" class="ui-tag-block"><span class="ui-tag">Podcast</span></a>
+      </div>
+      <div class="flex-grid-item flex-item-xxl-12 text-align-center pt-10 pt-l-7 pt-s-12">
+        <h1 class="u-visuallyhidden">Committed</h1>
+        <h3 class="u-visuallyhidden">Would you go to prison for your politics?</h3>
+        <div class="committed-archive__logo"></div>
+        <div class="committed-archive__subtitle "></div>
+      </div>
+    </div>
+  </section>
+</div>
+  <section class="container">
+    <div class="flex-grid-row mt-4 mb-4">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8">
+        <h3 class="mb-4 font-size-12 font-weight-bold">Listen to the trailer:</h3>
+        <iframe width="100%" height="166" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/1270698601&color=%23ffab70&auto_play=false&hide_related=true&show_comments=false&show_user=false&show_reposts=false&show_teaser=false"></iframe>
+      </div>
+    </div>
+    <div class="flex-grid-row mt-4 mb-5">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8 font-serif committed__serif-large text-paragraph-breaks">
+        <p>In the 1970s, the Provisional IRA was in the early days of its armed campaign to end British rule on the island of Ireland. In the United States, a small group of activists began organising on their behalf. They called themselves the Irish Northern Aid Committee, or Noraid—and they were looking for a fight.</p>
```

Comment:
This being the copy for Foreign Agent suggests not yet ready to merge? I think best to complete everything (or make this copy user editable via the CMS) before merging

Reply:
Yes, i understand but as we will need to go live with this at a later date I wanted you to have looked at the code. I have a list of all the things to edit before it goes to the actual site but we don't have the real copy or links yet